site/assignments/tic-tac-toe/tic-tac-toe.rst

```diff
@@ -1,6 +1,165 @@
-*******************************************
-I'm only taking CS to make sick video games
-*******************************************
+***********
+Tic-Tac-Toe
+***********
+
+* **Worth**: 10%
+* **Due**: TBD at 11:55pm; submitted on MOODLE
+* **Files**:
+
+
+Task
+====
+
+
+Provided Files
+==============
+
+
+Part 0 --- Read the Assignment
+==============================
+
+
+Part 1 --- Creating a new Notebook on Colab
+===========================================
+
+
+Part 2 --- Setup Game
+=====================
+
+
+Part 3 --- Parse Move
+=====================
+
+
+Part 4 --- Validate Move
+========================
+
+
+Part 5 --- Apply Move
+=====================
+
+
+Part 6 --- Check For Winner
+===========================
+
+
+Check Row & Column
+------------------
+
+
+Check Diagonals
+---------------
+
+
+Checking All Directions
+-----------------------
+
+
+Part 7 --- Rendering the Game Board
+===================================
+
+
+Render Cell
+-----------
+
+
+Render Row
+----------
+
+
+Render Board
+------------
+
+
+Part 8 --- Putting it Together
+==============================
+
+
+
+Part 9 --- Testing
+==================
+
+
+
+
+Some Hints
+==========
+
+* Work on one function at a time
+* Get each function working perfectly before you go on to the next one
+* Test each function as you write it
+    * This is a really nice thing about programming; you can call your functions and see what result gets returned
+    * Mentally test before you even write --- what does this function do? What problem is it solving?
+
+* If you need help, ask
+    * Drop by office hours
+
+
+Some Marking Details
+====================
+
+.. warning::
+    Just because your program produces the correct output, that does not necessarily mean that you will get perfect, or
+    even that your program is correct.
+
+Below is a list of both *quantitative* and *qualitative* things we will look for:
+
+* Correctness?
+* Did you follow instructions?
+* Comments?
+* Variable Names?
+* Style?
+* Did you do just weird things that make no sense?
+
+
+What to Submit to Moodle
+========================
+
+* Make sure your **NAME** and **STUDENT NUMBER** appear in a comment at the top of the program
+* Submit your version of ``asn3.py`` to Moodle
+    * Do **not** submit the .ipynb file
+    * To get the ``asn3.py`` file from Colab, see the example image in Assignment 1
+
+
+.. warning::
+
+    Verify that your submission to Moodle worked. If you submit incorrectly, you will get a 0.
+
+
+General FAQ:
+============
```

Comment:
Obligatory "this needs updating." The more I see it, the more I get towards the opinion of the wording here just being rude.

Reply:
I agree. 

It was written a few years ago and it was _fun_ and _hip_, but now as I get older, its turning from fun to _wow, what a dick_